packages/docs/website/package.json

```diff
@@ -1,14 +1,40 @@
 {
   "scripts": {
-    "examples": "docusaurus-examples",
-    "start": "docusaurus-start",
     "build": "docusaurus-build",
+    "examples": "docusaurus-examples",
+    "lint:js": "eslint '{,!(node_modules)/**/}*.js'",
```

Comment:
might be cleaner and more scalable to add an `.eslintignore` instead (I did this in `packages/ui` because there were several folders in the root that should be ignored): https://eslint.org/docs/user-guide/configuring#eslintignore

then the script can simply be:
"lint:js": "eslint **/*.js'",

Reply:
yes will fix